ispdb/t-online.de.xml

```diff
@@ -33,11 +33,11 @@
       <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
     </outgoingServer>
-    <enable visiturl="https://kundencenter.telekom.de/kundencenter/registriert/dienste-abos/ngcs/email-einstellungen/index.html">
-      <instruction>You need to set up an E-Mail-Password. Attention: The normal password for the T-Online-Website and Webmail does *not* work for email aplications.</instruction>
+    <enable visiturl="https://www.telekom.de/hilfe/apps-dienste/e-mail/programme/passwort-verwalten">
+      <instruction>You need to set up an E-Mail-Password. Attention: The normal password for the T-Online-Website and Webmail does *not* work for email applications.</instruction>
       <instruction lang="de">Sie müssen ein E-Mail-Passwort einrichten. Achtung: Das Passwort für die T-Online-Website und Webmail funktioniert *nicht* für E-Mail-Programme!</instruction>
     </enable>
-    <documentation url="http://hilfe.telekom.de/hsp/cms/content/HSP/de/3378/theme-45858870/theme-45858718/theme-45858716/theme-45858715/faq-45855229">
+    <documentation url="https://meinkonto.telekom-dienste.de/telekom/email/registration/registration.xhtml">
```

Comment:
This should contain a link to the documentation listing the mail server settings. So probably <https://www.telekom.de/hilfe/apps-dienste/e-mail/einstellungen/posteingang-postausgang-server>.

I don't think any clients are actually using this. The main purpose is to help with maintaining this database.
So instead of updating the `descr` elements, I suggest removing them.

Reply:
I addressed your suggestions. :)